svg/painting/parsing/stroke-dasharray-computed.svg

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     xmlns:h="http://www.w3.org/1999/xhtml"
+     width="800px" height="800px">
+  <title>SVG Geometry Properties: getComputedValue().strokeDasharray</title>
+  <metadata>
+    <h:link rel="help" href="https://svgwg.org/svg2-draft/painting.html#StrokeDasharrayProperty"/>
+    <h:meta name="assert" content="stroke-dasharray computed value uses absolute lengths.">
+  </metadata>
+  <g id="target"></g>
+  <style>
+    #target {
+      font-size: 40px;
+    }
+  </style>
+  <h:script src="/resources/testharness.js"/>
+  <h:script src="/resources/testharnessreport.js"/>
+  <h:script src="/css/support/computed-testcommon.js"/>
+  <script><![CDATA[
+
+test_computed_value("stroke-dasharray", "none");
+
+test_computed_value("stroke-dasharray", "10", "10px");
+test_computed_value("stroke-dasharray", "calc(10px + 0.5em)", "30px");
+test_computed_value("stroke-dasharray", "calc(10px - 0.5em)", "0px");
+test_computed_value("stroke-dasharray", "40%");
+test_computed_value("stroke-dasharray", "calc(50% + 60px)");
+
+test_computed_value("stroke-dasharray", "10px, 20%, 30px");
+test_computed_value("stroke-dasharray", "0, 5", "0px, 5px");
```

Comment:
Do we have a canonical rule about using comma separators vs space separators?

(I would have expected space to be the canonical version, myself.) Either way, we should test both.

Reply:
https://drafts.csswg.org/css-values-4/#component-multipliers
> A hash mark (#) indicates that the preceding type, word, or group occurs one or more times, separated by comma tokens 

https://drafts.csswg.org/cssom/#common-serializing-idioms
> To serialize a comma-separated list concatenate all items of the list in list order while separating them by ", ", i.e., COMMA (U+002C) followed by a single SPACE (U+0020).

I have added a test to stroke-dasharray-computed.svg (this PR) and stroke-dasharray-valid.svg (https://github.com/web-platform-tests/wpt/pull/14367)

Reply:
It's a bit more complicated than that with SVG, because most properties allow either comma or space separation, with no distinction in meaning.  SVG 2 uses the `<value>#*` combination list operator to describe it (a space-separated list of comma-separated lists), but that is slightly abusing the CSS syntax.

In this particular case, comma separation makes sense since it's an arbitrary-length list of similar values. And all browsers I tested did it that way, anyway. If we ever make `viewBox` or `points` or other attributes into CSS properties, then we'll have to deal with whether we want to enforce logic on the list separators!

Thanks for adding the extra test for space-separated values.